prepare.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Is the trailing `-` here by accident?

Reply:
No accident — it means "use empty value if unset". See [here](https://www.tldp.org/LDP/abs/html/refcards.html#AEN22728).